### PR TITLE
File Management and Organization

### PR DESCRIPTION
**Pull request by michaelj916**
As this site grows in complexity it might be useful to organize things into folders. I was considering adding changes to some of the pages styles since some are pretty blank, but I will leave this up to others unless told to do so.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -144,7 +146,7 @@ <h3>How to join the cheapETH movement</h3>
 <a href="https://discord.gg/r3WUGxzUH8" target="_blank">Join the discord</a><br />
 <a href="https://github.com/geohot/go-ethereum/tree/cheapeth" target="_blank">See the code</a> (we hired geohot to write it)<br />
 geth node setup <a href="pages/setup.html">instructions here</a><br />
-cheapETH <a href="https://twitter.com/CheapETH" target="_blank">twitter</a>
+cheapETH <a href="https://twitter.com/CheapETH">twitter</a>
```
Comment (michaelj916) [inline index.html]: This was a mistake, I was adding target="_blank" on other anchors but got rid of them all, and accidently removed this one as well.

**Comment by r0ck3tz** [comment]
We shouldn't accept some cheap centralized crap to our cheapETH network.

**Comment by rigwars** [comment]
LGTM!

**Review by c-z-chen — COMMENTED** [review]
yea it looks good but I don't know if people want the prior style—darker backgrounds and everything.

**Pull request closed without merging**